src/test/resources/application-test.yml

```diff
@@ -18,3 +18,6 @@ caserefgeneratorkey: rPr3qsjlU42hsDX5npfuCzlyF4UYfDTO
 queueconfig:
   consumers: 2 #this should be more than adequate to check for any multithreading problems
   retry-delay: 10  #this is for the tests around multiple receipts for the same case locking
+  outbound-uac-queue: case.rh.uac
+  outbound-case-queue: case.rh.case
+  outbound-printer-queue: Action.Printer
```

Comment:
these three lines don't belong in here. The `application-test.yml` only exists to over-ride prod config with test config. Stuff that's test specific needs to live somewhere else. My suggestion would be a `TestConstants` class with a bunch of `public static final` values written in all-caps to make it clear they're constants.

Reply:
`outbound-printer-queue` is also a dupe of `print-queue` which already exists in `application.yml`... so you can see why we should keep these prod config yamls as simple as possible to stop the config becoming a confusing nightmare